tests/unit/javascript/stylesheet/test_stylesheet.py

```diff
@@ -0,0 +1,131 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Jay Kamat
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for stylesheet.js."""
+
+import os
+import pytest
+
+try:
+    from PyQt5.QtWebEngineWidgets import QWebEngineProfile
+except ImportError:
+    QWebEngineProfile = None
+
+
+from qutebrowser.utils import javascript
+from qutebrowser.browser.webengine import webenginesettings
+
+
+DEFAULT_BODY_BG = "rgba(0, 0, 0, 0)"
+GREEN_BODY_BG = "rgb(0, 255, 0)"
+CSS_BODY_GREEN = "body {background-color: rgb(0, 255, 0);}"
+CSS_BODY_RED = "body {background-color: rgb(255, 0, 0);}"
+
+
+class StylesheetTester:
+
+    """Helper class (for the stylesheet_tester fixture) for asserts.
+
+    Attributes:
+        js: The js_tester fixture.
+        config_stub: The config stub object.
+    """
+
+    def __init__(self, js_tester, config_stub):
+        self.js = js_tester
+        self.config_stub = config_stub
+
+    def init_stylesheet(self, css_file="green.css"):
+        """Initialize the stylesheet with a provided css file."""
+        css_path = os.path.join(os.path.dirname(__file__), css_file)
+        self.config_stub.val.content.user_stylesheets = css_path
+        if QWebEngineProfile is None:
+            pytest.skip("QTWebEngine not found.")
```

Comment:
Is this fine for skipping QtWebEngine tests, or is there a special attribute or similar I can use instead?

Reply:
Shouldn't be needed anymore when using `importorskip` above.